frontends/client/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "trieve-fetch-client",
+  "type": "module",
+  "exports": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "version": "0.0.1",
```

This should match the OpenAPI spec version

```suggestion
  "version": "0.11.7",
```